src/infix-longscale.ts

```diff
@@ -0,0 +1,52 @@
+
+// Copy-pasted from standard.ts
+const ABBREVIATIONS = [
+  ",", "K", "M", ".", "B", ".", "T", ".", "Qa", ".", "Qt", ".", "Sx", ".",
+  "Sp", ".", "Oc", ".", "No", ".", "Dc", ".", "UDc", ".", "DDc",
+  "TDc", ".", "QaDc", ".", "QtDc", ".", "SxDc", ".", "SpDc", ".", "ODc", ".",
+  "NDc", ".", "Vg", ".", "UVg", ".", "DVg", ".", "TVg",
+  "QaVg", ".", "QtVg", ".", "SxVg", ".", "SpVg", ".", "OVg", ".", "NVg", ".",
+  "Tg", ".", "UTg", ".", "DTg", ".", "TTg", ".", "QaTg",
+  "QtTg", ".", "SxTg", ".", "SpTg", ".", "OTg", ".", "NTg", ".", "Qd", ".",
+  "UQd", ".", "DQd", ".", "TQd", ".", "QaQd", ".", "QtQd",
+  "SxQd", ".", "SpQd", ".", "OQd", ".", "NQd", ".", "Qi", ".", "UQi", ".",
+  "DQi", ".", "TQi", ".", "QaQi", ".", "QtQi", ".", "SxQi",
+  "SpQi", ".", "OQi", ".", "NQi", ".", "Se", ".", "USe", ".", "DSe", ".",
+  "TSe", ".", "QaSe", ".", "QtSe", ".", "SxSe", ".", "SpSe",
+  "OSe", ".", "NSe", ".", "St", ".", "USt", ".", "DSt", ".", "TSt", ".",
+  "QaSt", ".", "QtSt", ".", "SxSt", ".", "SpSt", ".", "OSt",
+  "NSt", ".", "Og", ".", "UOg", ".", "DOg", ".", "TOg", ".", "QaOg", ".",
+  "QtOg", ".", "SxOg", ".", "SpOg", ".", "OOg", ".", "NOg",
+  "Nn", ".", "UNn", ".", "DNn", ".", "TNn", ".", "QaNn", ".", "QtNn", ".",
+  "SxNn", ".", "SpNn", ".", "ONn", ".", "NNn", ".", "Ce"
+];
+
+
+import { AbstractInfixNotation } from "./infix-abstract";
+import Decimal from "break_infinity.js";
+
+// Name comes from https://en.wikipedia.org/wiki/Long_and_short_scales
+export class InfixLongScaleNotation extends AbstractInfixNotation {
+  public get name(): string {
+    return "Infix long scale";
+  }
+
+  public formatDecimal(value: Decimal, places: number): string {
+    let str = super.formatDecimal(value, Math.max(places, 5));
```

Comment:
Is there a reason to uniquely use 5 places at most here?

Reply:
OK, I see, 5 places at least. That's somewhat more explicable. I'll try the notations out and let you know what I think.

Reply:
Hm, I see that all the others do 2 places at least as part of AbstractInfixNotation. Makes sense then, I guess.

Reply:
The rationale is that long-scale formatting should always display at least one separator, so I'm forcing the code to display at least 6 significant digits (which translates to 5 "decimal places")